File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/ContextFilters.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class ContextFilters(
  val include: List<CodyContextFilterItem>? = null,
  val exclude: List<CodyContextFilterItem>? = null,
)

File: agent/bindings/csharp/ContextFilters.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class ContextFilters
  {
    [JsonProperty(PropertyName = "include")]
    public CodyContextFilterItem[] Include { get; set; }
    [JsonProperty(PropertyName = "exclude")]
    public CodyContextFilterItem[] Exclude { get; set; }
  }
}
